project/LinkerdBuild.scala

```diff
@@ -258,7 +258,12 @@ object LinkerdBuild extends Base {
        |   -Dio.netty.allocator.numDirectArenas=${FINAGLE_WORKERS:-8}    \
        |   -Dcom.twitter.finagle.netty4.numWorkers=${FINAGLE_WORKERS:-8} \
        |   ${GC_LOG_OPTION:-}                                            \
-       |   ${LOCAL_JVM_OPTIONS:-}                                        "
+       |   ${LOCAL_JVM_OPTIONS:-}"
+       |
+       |if ! [ "$LOCAL_JAVA_VERSION" -ge 9 ]; then
```

Comment:
maybe just me, but I find having an if statement here to be a bit unwieldy.  what about just having a `${GC_OPTION:-}` here and populating that based on version, just like we do with `${GC_LOG_OPTION:-}`?